pages.pt-BR/common/7za.md

```diff
@@ -0,0 +1,25 @@
+# 7za
+
+> Um compactador de arquivos com alta taxa de compressão.
+> Versão do `7z` com suporte para poucos tipos de arquivamento/compressão.
```

Comment:
IMO "fewer" should translate to "menos", not "poucos". Otherwise it's unclear what we're measuring against. WDYT @andrik, @schneiderl?

Reply:
What about?

> Versão com menos opções de tipos de arquivamento/compressão em relação ao `7z` .

Reply:
Even better! :D

Reply:
:+1: